src/ui/pages/article/EditArticlePage.js

```diff
@@ -0,0 +1,75 @@
+import { expect, test } from '@playwright/test';
+
+export class EditArticlePage {
+  constructor(page) {
+    this.page = page;
+    this.editPage = page.getByRole('link', {name: 'Edit Article'});
+    this.titleField = page.getByPlaceholder('Article Title');
+    this.descriptionField = page.getByPlaceholder(`What's this article about?`);
+    this.textField = page.getByPlaceholder('Write your article (in markdown)');
+    this.tagField = page.getByPlaceholder('Enter tags');
+    this.updateArticleButton = page.getByRole('button', {name: 'Update Article'});
+  }
+
+  async goToEditArticlePage() {
+    await test.step('go to edit article page', async () => {
+      await this.editPage.nth(1).click();
+    });
+  };
+
+  async editArticleTitleField(title) {
```

Comment:
```suggestion
  async fillArticleTitle(title) {
```
you name confuses a little bit, because I think that you have additional logic inside but you just fill the field